### PR TITLE
Since Meta doesn't seem to want 2d apps yet we'll remove this build.

### PR DESCRIPTION
**Pull request by JBassett**
## Summary

This build was actually slowing down PR and release build times.

## Link to pull request in Documentation repository

N/A

### CODE REVIEW

**Review by dshokouhi — APPROVED** [review]

**Comment by dshokouhi** [comment]
To any other user reading this, this does not impact the listing on Sidequest as that uses the minimal build

**Comment by jpelgrom** [comment]
I think README.md should also be updated to remove quest references

**Comment by JBassett** [comment]
> I think README.md should also be updated to remove quest references

Missed those references, thanks!

File: README.md [hunk 1]
```diff
@@ -16,8 +16,6 @@ If you are looking for documentation around the companion applications check out
  - `io.homeassistant.companion.android.debug`
  - `io.homeassistant.companion.android.minimal`
  - `io.homeassistant.companion.android.minimal.debug`
- - `io.homeassistant.companion.android.quest`
- - `io.homeassistant.companion.android.quest.debug`
 
 5. Now download the `google-services.json` file and put it in the _home-assistant-Android/app_ folder. This file contains the configuration of the whole project (all the six applications). ([You can also use the mock services file instead of generating your own](/.github/mock-google-services.json). The file should contain client IDs for all packages listed above for debugging to work properly.  **If you do not generate your own file push notification will never work**)
```
Comment (dshokouhi) [inline README.md:20]: Should probably update to say `four` :) same with step 4 (and yes I missed that earlier myself when I updated the readme for quest builds)

**Pull request merged**